roles/apache/templates/apache_vhost.conf.j2

```diff
@@ -22,13 +30,63 @@
     SSLProtocol all -SSLv2 -SSLv3
 {# 15768000 seconds = 6 months #}
     Header always set Strict-Transport-Security max-age=15768000
-    {% if item.signedcertificate is defined and item.signedcertificate %}
+    {% if item.ssl.selfsignedcertificate is defined and item.ssl.selfsignedcertificate %}
     SSLUseStapling on
     {% else %}
     SSLUseStapling off
     {% endif %}
 {% endif %}
-
-    # VHost specific config
-{{ item.config }}
+{% if proxy %}
+  {% if 'sympa' in item %}
+    <Location /sympa>
+              Require all granted
+              RequestHeader set Front-End-Https "On"
+              ProxyPass         https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/sympa
+              ProxyPassReverse  https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/sympa
+              ProxyPreserveHost On
+    </Location>
+    <Location /sympasoap>
+              Require all granted
+              RequestHeader set Front-End-Https "On"
+              ProxyPass         https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/sympasoap
+              ProxyPassReverse  https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/sympasoap
+              ProxyPreserveHost On
+    </Location>
+    <Location /check-smtp-service-results>
+              Require all granted
+              RequestHeader set Front-End-Https "On"
+              ProxyPass         https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/check-smtp-service-results
+              ProxyPassReverse  https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/check-smtp-service-results
+              ProxyPreserveHost On
+    </Location>
+    <Location /static-sympa>
+              Require all granted
+              RequestHeader set Front-End-Https "On"
+              ProxyPass         https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/static-sympa
+              ProxyPassReverse  https://{{ hostvars[item.sympa.server]['ansible_eth1']['ipv4']['address'] }}/static-sympa
+              ProxyPreserveHost On
+    </Location>
+  {% endif %}
+  {% if 'dokuwiki' in item %}
```

Comment:
why is there a doku wiki and limewire entry in here?

Reply:
Missing cleanup. sorry. It should not. I'll remove it.
But as an explanation: because in Universalistes (groupes.renater.fr) we use dokuwiki and limesurvey as Sympa subrogates.

Reply:
i figured :D

Reply:
This was fixed in commit 8ba22d0.
I was suspecting it for a while but I'm pretty sure now: in order to have playbooks working in a multi-tools environment, we need to get rid of templates in most cases and edit parts of the config files from tasks, using lineinfile and blockinfile.

Reply:
> This was fixed in commit [8ba22d0](https://github.com/sympa-community/sympa-ansible/commit/8ba22d093ffe124c84d0182c439ebd620500183e).
> I was suspecting it for a while but I'm pretty sure now: in order to have playbooks working in a multi-tools environment, we need to get rid of templates in most cases and edit parts of the config files from tasks, using lineinfile and blockinfile.

Not necessarily - many servers provide include statements in their configuration set which can solve this in some cases.